app/services/katello/registration_manager.rb

```diff
@@ -266,6 +268,14 @@ def pulp_consumer_destory(host_uuid)
         Rails.logger.warn(_("Pulp Consumer %s has already been removed") % host_uuid)
       end
 
+      def delete_agent_queue(host)
+        queue_name = Katello::Agent::Dispatcher.host_queue_name(host)
```

Comment:
Since the host record (including content facet) will be gone by the time we issue the queue delete to Qpid, we need a way to derive what its queue name would be, and that's what this method does.